dom.bs

```diff
@@ -2637,11 +2635,12 @@ To <dfn export for=Node id=concept-node-replace-all>replace all</dfn> with a
  <li>Let <var>removedNodes</var> be <var>parent</var>'s
  <a>children</a>.
 
- <li>Let <var>addedNodes</var> be the empty list if <var>node</var> is
- null, <var>node</var>'s <a>children</a> if
- <var>node</var> is a {{DocumentFragment}}
- <a>node</a>, and a list containing <var>node</var>
- otherwise.
+ <li><p>Let <var>addedNodes</var> be the empty list.
+
+ <li><p>If <var>node</var> is {{DocumentFragment}} <a>node</a>, then set <var>addedNodes</var> to
+ <var>node</var>'s <a>children</a>.
+
+ <li><p>Otherwise, if <var>node</var> is non-null, set <var>addedNodes</var> to « <var>node</var> ».
```

Comment:
@domenic this is the only change that felt a little weird here due to it getting so much longer, but it's probably better?

Reply:
Yeah, this matches the code better. Nested ternaries are not great in either English or code, IMO. You could also use dl class="switch".